task1/task1_1.js

```diff
@@ -0,0 +1,14 @@
+import { Transform } from 'stream';
+
+class ReverseTransform extends Transform {
+  _transform(buf, encoding, callback) {
+    const inputStr = buf.toString('utf-8');
+    const reversedStr = inputStr.trim().split('').reverse().join('');
+
+    callback(null, reversedStr + '\n');
+  }
```

Comment:
pls use
`try {
            callback(null, chunk)
        } catch (err) {
            callback(err);
        }`
to pass an error further